classes/py1.html

```diff
@@ -6,9 +6,9 @@
 
 <div class="py1-wrapper">
     <div class="py1-sidebar">
-        <h1>how to get started in<br />
-        <span class="py1-link">python</span>
-        </h1>
+        <span class = "py1-smaller-margin-p">how to get started in</span>
+        <span class= "py1-smaller-margin-heading"><span class = "py1-link">python</span></span>
```

Comment:
A minor piece of feedback, but I think it's a bit confusing to use our link styling if this thing isn't actually a link. If we want the underline effect (without the hover), we can implement that independently; however, it might also be worth picking a style that doesn't clash with our link styling.

Reply:
I tried a few different options and ended up deciding to leave out any effects here. With the added image directly below the title, I thought that it looked cleaner without anything, but am open to suggestions if you or @EinarBalan think including some sort of effect would look nicer!